lib/options.js

```diff
@@ -124,6 +124,9 @@ function assignPublishing(opts) {
 
   if (opts.ably && opts.channel) {
     try {
+      opts.ably = opts.ably === true ? {} : opts.ably || {};
+      opts.ably.log = { level: 0 };
+
```

Comment:
Good call on opts.ably already being known to be true and deoptimization. However I wouldn't want to use the assign because no matter what we want the log level to be 0.

Now that I think about it more I'm just not going to make the options configurable at all via the API.